### DISCUSSION TITLE
GitHub Discussions are now read-only – please join us in the forum!

### DISCUSSION THREAD

**Discussion by carolinefrasca** [modular/modular#3554] [Announcements]
As part of our efforts to centralize our community and answer questions as quickly as possible, GitHub Discussions on this repository are now read-only. Please share your questions, thoughts, and projects in the [Modular forum](https://forum.modular.com/) instead.

Thank you for being part of our community and see you in the forum!

**Comment by felixfeixu** [comment]
I know this is decided, but discussion is more topic oriented and easier to browse by categories/tags.

**Reply by carolinefrasca** [reply]
Hi @felixf4xu, thank you for the feedback! I'd recommend checking out [our new forum](https://forum.modular.com/), which is structured similarly.

**Comment by remorses** [comment]
Who decided this? Do you think there are more programmers on GitHub (and searching Google) or Discord (a chat app for gamers and teenagers)?

**Reply by iamtimdavis** [reply]
We now have the [Modular Forum](https://forum.modular.com/) if you'd like to use that instead!

**Comment by geet-b** [comment]
👍🏻
